VenafiPS/Public/Invoke-VenafiCertificateAction.ps1

```diff
@@ -271,7 +271,7 @@ function Invoke-VenafiCertificateAction {
 
         if ( $performInvoke ) {
             try {
-                Invoke-VenafiRestMethod @params -FullResponse -Verbose | Out-Null
+                Invoke-VenafiRestMethod @params -FullResponse | Out-Null
```

Comment:
```suggestion
                $null = Invoke-VenafiRestMethod @params -FullResponse
```
we should move to null assignment instead of out-null for better performance

Reply:
Works for me. I'll make those changes.